packages/react-native-editor/ios/Podfile

```diff
@@ -39,7 +39,7 @@ target 'gutenberg' do
   pod 'Folly', :podspec => '../../../node_modules/react-native/third-party-podspecs/Folly.podspec'
   pod 'RNTAztecView', :path => '../../react-native-aztec/RNTAztecView.podspec'
   #Use for testing non official versions of Aztec
-  pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/WordPress-Aztec-iOS.git', :commit => '9187001a9c6dc84c211937b1b5a2af2a8b8d54a9'
+  pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/WordPress-Aztec-iOS.git', :tag => '1.19.3'
```

Comment:
You only this reference when testing the changes in Aztec, the moment you update the dependency on the RNTAztecView.podspec file you should comment this line.